kratos/python/add_amgcl_solver_to_python.cpp

```diff
@@ -48,6 +48,7 @@ void  AddAMGCLSolverToPython(pybind11::module& m)
 
     enum_<AMGCLSmoother>(m,"AMGCLSmoother")
     .value("SPAI0", SPAI0)
+    .value("SPAI1", SPAI1)
```

Comment:
i actually would like to remove this construction mode and only leave the one by parameters...

in any case let's remove it at a later time